src/pages/Address/AddressAssetComp.tsx

```diff
@@ -192,18 +232,27 @@ export const AddressCoTAComp = ({ account }: { account: CoTA }) => {
   )
 }
 
-export const AddressOmigaInscriptionComp = ({ account }: { account: OmigaInscription }) => {
+export const AddressOmigaInscriptionComp = ({
+  account,
+  isRGBPP,
+  udtLabel,
```

Comment:
It's `omiga inscription comp` so the label should be `omiga symbol`

Reply:
done